components/dashboard/src/Menu.tsx

```diff
@@ -80,6 +80,10 @@ export default function Menu() {
     const leftMenu: Entry[] = (() => {
         if (!team) {
             return [
+                ...(showTeamsUI ? [{
+                    title: 'Projects',
+                    link: '/projects'
```

Comment:
**question**: What do you think of defaulting to the first tab (projects) also for personal accounts? See also [relevant comment](https://github.com/gitpod-io/gitpod/issues/4920#issuecomment-886538402).

Reply:
This is a bit tricky to achieve, because this changes the default tab for individuals based on whether the teams UI is enabled or not.

I'm not against such a change, but maybe it's safer to wait for the Teams & Projects UI to be generally enabled for everyone.

Reply:
DEAL. Let's ship this and see what we can change later. ➿

Reply:
FYI Added follow up issue for this in https://github.com/gitpod-io/gitpod/issues/5050. Cc @jankeromnes @AlexTugarev @svenefftinge

Reply:
Thank you so much for not letting this slip by @gtsiolis! 💯 💯 💯